neurons/validator.py

```diff
@@ -286,6 +286,13 @@ async def handle_synthetic_patch_response(
             TaskType.LABELLED_ISSUE
         )
 
+        for miner_hotkey, grade in zip(miner_hotkeys, rewards_list):
+            self.logger.info(f"Graded miner {miner_hotkey} with score of {grade} for question {problem_uuid}", extra=asdict(LogContext(
+                log_type="lifecycle",
+                event_type="solution_selected",
+                additional_properties={"question_id": problem_uuid, "grade": grade, "miner_hotkey": miner_hotkey}
+            )))
+
```

Comment:
This should go here: https://github.com/taoagents/agentao/blob/7ad1ccf37e69a1d0943fe359c7858cb4fb84264f/agentao/validator/graders/float_grader.py#L127